proposals/lockout-violation-detection.md

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
Filename should be `0009-lockout-violation-detection.md`

Reply:
resolved